Rakefile

```diff
@@ -79,6 +79,33 @@ task :'release:docs' => :docs do
   sh "aws s3 cp --recursive docs/_build/html/ s3://#{S3_BUCKET}/#{S3_DIR}/docs/"
 end
 
+namespace :pypi do
+  RELEASE_DIR = '/tmp/dd-trace-py-release'
+
+  task :clean do
+    FileUtils.rm_rf(RELEASE_DIR)
+  end
+
+  task :build => :clean do
+    puts "building release in #{RELEASE_DIR}"
+    sh "python setup.py -q sdist -d #{RELEASE_DIR}"
+  end
+
+  task :release => :build do
```

Comment:
the ``release:wheel`` task should not be used in the [circle.yml file][1]. We can remove that line so only docs are built when a tag is pushed (we will move docs later). For the ``master`` and ``develop`` branch is good to keep it, so we can publish intermediate wheel packages for internal testing.

[1]: https://github.com/DataDog/dd-trace-py/blob/master/circle.yml#L39